numba/typed/typedlist.py

```diff
@@ -299,6 +304,16 @@ def copy(self):
     def index(self, item, start=None, stop=None):
         return _index(self, item, start, stop)
 
+    def sort(self, key=None, reverse=False):
+        """Sort the list inplace.
+
+        See also ``list.sort()``
+        """
+        # If key is not already a dispatcher object, make it so
```

Comment:
This ought to work? But the key isn't a dispatcher type.
```python
from numba import njit
import numpy as np
from numba.typed import List

l = List()
[l.append(x) for x in range(10)]

key = njit(lambda x: 1./x)
l.sort(reverse = True, key=key)
print(l)
```

Reply:
I mixed up the `numba.types.Dispatcher` and `numba.dispatcher.Dispatcher`.

Reply:
fixed now